content/en/registry/sdk-extension-python-aws.md

```diff
@@ -0,0 +1,14 @@
+---
+title: AWS SDK Extension
+registryType: sdk-extension
```

Comment:
Could this be changed to 'utilities'? We don't have a mapping for sdk-extension so it breaks the filters/labels.

Reply:
I've changed it to be `utilities`! I think that's a good enough label of what it is 👍